lib/fluent/plugin/elasticsearch_error_handler.rb

```diff
@@ -23,6 +23,10 @@ def unrecoverable_error?(type)
     unrecoverable_error_types.include?(type)
   end
 
+  def unrecoverable_record_error?(type)
+    ['json_parse_exception'].include?(type)
```

Comment:
It's pretty good for adding unrecoverable record error concept, but I think that `es_rejected_execution_exception` is also unrecoverable exception.

What do you think about this?
Keep as-is for now is also OK for me.

Reply:
I defer to you here as I do not know.  If this is an exception one might expect from trying to push an individual record and it is not possible to recover or not possible to successfully resubmit, I agree we can add this to the list